src/pages/TestingModelPage.tsx

```diff
@@ -45,6 +47,18 @@ const TestingModelPage = () => {
     }
   }, [model, navigateToDataSamples]);
 
+  const tourStart = useStore((s) => s.tourStart);
```

Comment:
I'd like a better plan but I think that involves teaching the store about connections and routes.